src/Common/src/TypeSystem/IL/Stubs/ILEmitter.cs

```diff
@@ -31,10 +31,13 @@ public LabelAndOffset(ILCodeLabel label, int offset)
 
         private ArrayBuilder<LabelAndOffset> _offsetsNeedingPatching;
 
-        internal ILCodeStream()
+        private ILEmitter _emitter;
```

Comment:
I think this is a good change (making `ILCodeStream` offer more "high level" API surface will help with fixing #2119), but cc @jkotas just in case he had different plans.

(The immediate problem at hand here could also be solved with extension methods after all.)